core-helm-chart/templates/deployment.yaml

```diff
@@ -15,37 +15,106 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Values.prefix }}-server
-  namespace: {{ .Values.namespace }}
+  name: kubestellar
 spec:
   selector:
     matchLabels:
-      app: {{ .Values.prefix }}-server
+      app: kubestellar
   replicas: 1
   template:
     metadata:
       labels:
-        app: {{ .Values.prefix }}-server
+        app: kubestellar
     spec:
-      serviceAccountName: {{ .Values.prefix }}-sa
+      serviceAccountName: kubestellar-sa
       containers:
-      - name: {{ .Values.prefix }}-server
+
+      - name: kcp
+        securityContext:
+          runAsNonRoot: true
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop: ["ALL"]
+          seccompProfile:
+            type: RuntimeDefault
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        # command: [ "sleep" , "infinity" ]
+        command: [ "/home/kubestellar/entry.sh" , "kcp" ]
         env:
```

Comment:
Can we get verbosity control here too?

Reply:
@MikeSpreitzer can you please helm me by letting me know the verbosity argument for kcp start?
Also would the "value" be the same than the others.
For simplicity I'm using a common one at the moment.

Reply:
`kcp start` takes the usual verbosity argument, `-v`.
I think it makes sense to give individualized verbosity control, not force the same verbosity level to apply to all four processes.
It might be good to frame this such that if the user does not override any values then none of the processes is given a `-v` (each applies its own default), but there are four values that the user can optionally set to cause a specific `-v=N` to be given to a process.

None of this verbosity stuff is a blocker for this PR; we can do more in follow-ons.

Reply:
@MikeSpreitzer I will implement in a separated PR. I'm concerned to mess this one up.